governance.md

```diff
@@ -109,6 +109,16 @@ This is the purpose of Working Groups (WG). The intent is to make
 Working Groups relatively easy to create and to deprecate, once
 inactive.
 
+If you would like to propose a new working group, send a proposal to
+kubernetes-dev@googlegroups.com and also include any potentially 
+interested SIGs. Wait for public comment. If there's enough interest,
+a new Working Group should be formed. 
+
+Create a new mailing list in the from of kubernetes-wg-group-name.
+Working groups typically have a Slack channel as well as regular meetings. 
+It's encouraged to keep a clear record of all accomplishments that's 
+publicly accessible.
+
```

Comment:
Typically we have required SIGs to sponsor working groups. Do we require a SIG to sponsor a working group, or can a WG be formed independently of any SIG?

Reply:
I asked Kim to update with the SIG sponsor info (as well as some other things). Looking back on several previous WG creations, there was at least one sponsoring sig pre-proposal.